README.md

```diff
@@ -201,6 +201,7 @@ models in your framework so you can store data offline with localForage. We
 have drivers for the following frameworks:
 
 * [AngularJS](https://github.com/ocombe/angular-localForage)
+* [Angular](https://www.npmjs.com/package/ngforage)
```

Comment:
I think this should say `AngularJS 4+` or something like that, to differentiate.

Reply:
@tofumatt b3013f901907b1b92efea8f9eafa6faa7d6ce756 :slightly_smiling_face:

Reply:
I stuck to Angular vs AngularJS as AngularJS is widely known to refer to 1.x whilst Angular is known to refer to 2.x+

Reply:
Ah okay, cool. I wasn't sure. 😄